group_vars/all.yml

```diff
@@ -220,9 +220,9 @@ aws_tags_kms_database:
 # --------  AMI  --------
 # -----------------------
 
-# AMI Name search pattern matching official CentOS 7 Marketplace AMI
+# AMI Name search pattern matching official Rocky linux 9 Marketplace AMI
 # Warning! You need to be subscribed to it first!
-aws_ami_base_marketplace_name: "*b7ee8a69-ee97-4a49-9e68-afaee216db2e*"
+aws_ami_base_marketplace_name: "Rocky-9-EC2-*"
```

Comment:
Please remove the whole subscription logic. Too much fuzz with it. Let's get back to hardcoding ami-id instead and updating that periodically.